platforms/enterprise/enterprise/src/main/java/org/gradle/internal/enterprise/impl/GradleEnterprisePluginServices.java

```diff
@@ -30,8 +30,11 @@ public void registerBuildTreeServices(ServiceRegistration registration) {
         // legacy
         registration.add(DefaultBuildScanClock.class);
         registration.add(DefaultBuildScanBuildStartedTime.class);
+
         registration.add(GradleEnterpriseAutoAppliedPluginRegistry.class);
         registration.add(GradleEnterprisePluginAutoAppliedStatus.class);
+        registration.add(DefaultGradleEnterprisePluginServiceRef.class);
+        registration.add(DefaultGradleEnterprisePluginConfig.class);
```

Comment:
💭  Small nit: should we swap the legacy and non-legacy "sections" and have the legacy at the end?
This would be the same way it is for `registerGradleServices` now, and would make it clear where to append new services and avoid what Luke fixed with an empty line above.

Reply:
Makes sense, I reordered them here: https://github.com/gradle/gradle/pull/26082/commits/5eb4f519310039c751788470a5de4f967264c071#diff-53962f1c238b13fa46a822ee80987c835e7b12a0355af9c850c49c6a4a9864c2